<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Write</string>
   </property>
   <widget class="QTextEdit" name="write_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>411</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Read</string>
   </property>
   <widget class="QTextEdit" name="read_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>207</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Server IP：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="server_ip"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Port：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>132</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
